configuration/openmrs/apps/home/extension.json

```diff
@@ -78,5 +78,15 @@
     "icon": "fa fa-calendar",
     "order": 9,
     "requiredPrivilege": "app:appointments"
+  },
+  "implementerInterface": {
+    "id": "bahmni.implementer.interface",
+    "extensionPointId": "org.bahmni.home.dashboard",
+    "type": "link",
+    "translationKey": "MODULE_LABEL_IMPLEMENTER_INTERFACE_KEY",
```

Comment:
Could you provide this key translation in the `locale_fr.json`?

Reply:
This is done in below commit